addons/mail/static/src/models/user.js

```diff
@@ -70,7 +70,7 @@ registerModel({
          *
          * If a chat is not appropriate, a notification is displayed instead.
          *
-         * @returns {Thread|undefined}
+         * @returns {Channel|undefined}
          */
         async getChat() {
```

Comment:
**=== Out-of-scope of PR ===**
`getChat()` is badly coded. Should actually just be a field `chat`. Not trivially feasible just because it's `async` and does `fetch` + side-effects like triggering notification, so quite bad.